<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUdp</class>
 <widget class="QWidget" name="mainUdp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainUdp</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="pltMessageShow"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>对方IP地址:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ldtIpAdress"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>对方端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ldtIpPort"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnSend">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPlainTextEdit" name="pteSend"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
